src/Product/SearchProvider.php

```diff
@@ -82,6 +83,9 @@ private function getAvailableSortOrders()
         $translator = $this->module->getTranslator();
 
         return [
+            $sortSalesDesc->setLabel(
+                $translator->trans('Popularity', [], 'Modules.Facetedsearch.Shop')
```

Comment:
@Junebyun Wdyt about 'Popularity' ? Maybe 'bestsellers' is more explicit ?

Reply:
@marionf yes since it will be the label of a list sorting best sales, I think _Best sellers_ will be more accurate!

Reply:
@Junebyun, perhaps to be put in parallel with https://github.com/PrestaShop/PrestaShop/issues/19145. :-)